osf/migrations/0042_add_registration_registered_date_desc_index.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9 on 2017-03-09 20:37
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('osf', '0041_auto_20170308_1932'),
+    ]
+
+    operations = [
+        migrations.RunSQL(
+            [
+                'CREATE INDEX osf_abstractnode_registered_date_index ON public.osf_abstractnode (registered_date DESC);',
+                "CREATE INDEX osf_abstractnode_registration_pub_del_type_index ON public.osf_abstractnode (is_public, is_deleted, type) WHERE is_public=TRUE and is_deleted=FALSE and type = 'osf.registration';"
```

Comment:
@icereval Then I added these partial indexes on the things we filter on all the time because the query I'd written was doing a seq scan. This seems to have made things much faster. ~350ms -> 66ms. Hopefully the nodes one will help other things.